README.md

```diff
@@ -513,6 +513,8 @@ To enable usage of pre-installed `apollo-codegen` module, set gradle system prop
 systemProp.apollographql.useGlobalApolloCodegen=true
 ```
 
+Note that this requires exactly version `0.19.1` (not older, not newer). You can install this conventionall via `npm install -g apollo-codegen@0.19.1`.
```

Comment:
What does conventionall mean?

Reply:
a succinct, typo-friendly abbreviation for `conventionally`

6b402c579c5ba96937d9cd15b9c40b1975d6d1b3